packages/api/src/v1-queries/get-portfolio-positions/getV1PortfolioPositionDetails.ts

```diff
@@ -107,7 +108,7 @@ export const getV1PortfolioPositionDetails = async ({
       canSettle: true,
       rolloverPoolId,
 
-      history: txs,
+      history: includeHistory || response.pool.flags.isGLP28Jun2023 ? txs : [],
```

Comment:
Don't do the logic here. Why even bother to push items to txs if we dont need them? Save some CPU time man :) Here it should always be history: txs but on upper level you determine if array needs to be built!

Reply:
https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/126/commits/972f309bd5b012bea2cc64b798f4c754a2fdd8e4